cartograph/server/ParentService.py

```diff
@@ -23,13 +30,41 @@ def service_for_map(self, map_name):
         """
         return getattr(self.map_services[map_name], self.service_name)
 
+    def update_maps(self, meta_config):
+        """Initialize any map whose map-config is in the meta-config, but has not yet been initialized.
+        :return:
+        """
+        with open(meta_config, 'r') as configs:
+            configs.next()  # Skip the multi-map flag (i.e. '######')
+            for map_config in configs:
+                map_config_path = map_config.strip('\r\n')
+
+                # If the name of a map isn't in map_services, initialize it
+                config = Config.initConf(map_config_path)
+                config_name = config.get('DEFAULT', 'dataset')
+                if config_name not in self.map_services.keys():
+                    map_service = MapService(map_config_path)
+                    self.map_services[map_service.name] = map_service
+
+        # indicate that map_services has been updated
+        self.map_services['_last_update'] = os.stat(meta_config)
```

Comment:
Use mtime instead of all of os.stat. Os.stat includes access time, which may be updated without any real changes.